ui/core/src/model/calculations.ts

```diff
@@ -49,7 +49,7 @@ export const CALCULATIONS_CONFIG: Readonly<Record<CalculationType, CalculationCo
     description: 'Last numeric value',
   },
   Mean: {
-    label: 'Average',
+    label: 'Avg',
```

Comment:
Making this smaller to fit better in the headers now that we have the sort icon taking space. I also recall someone recommending this previously as a shortening of this value, but I forgot to set it this way when doing the calculation update PRs.